layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend

```diff
@@ -2,21 +2,18 @@ inherit kernel-resin
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+LINUX_VERSION = "4.19.44"
+
 SRC_URI_append = " \
-	file://0001-Revert-cgroup-Disable-cgroup-memory-by-default.patch \
 	file://0002-wireless-wext-Bring-back-ndo_do_ioctl-fallback.patch \
 	file://0003-leds-pca963x-Fix-MODE2-initialization.patch \
 	file://0001-Add-npe-x500-m3-overlay.patch \
 	file://0002-NFLX-2019-001-SACK-Panic.patch \
-	file://0003-NFLX-2019-001-SACK-Panic-for-lteq-4.14.patch \
```

Comment:
Could you please add the reason (in git log) why removed this patch?

Reply:
All Netflix patches are applied upstream now, so Andrei's comment is obsolete